sdk/keyvault/keyvault-admin/CHANGELOG.md

```diff
@@ -1,15 +1,19 @@
 # Release History
 
-## 4.4.0-beta.2 (Unreleased)
+## 4.4.0 (Unreleased)
```

Comment:
You should set a date, too. Did you run the _eng/common/scripts/Prepare-release.ps1_ script yet? You should, though it may be too late to get in on the release notes. That would set the date and get rid of empty sections below.

Reply:
I ran the script last week. I usually run the `Prepare-Release` script with the `-ReleaseTrackingOnly` option and run it again at a more convenient time without the flag to fix up the release notes. Usually I make the changes from the script in a separate PR just before I release, since I've had it once or twice where I want to add something else to the changelog later in a section that the script already deleted.